index.html

```diff
@@ -6,17 +6,60 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-    <title>My Blog</title>
     <link
       href="//fonts.googleapis.com/css?family=Roboto:400,500,300"
       rel="stylesheet"
       type="text/css"
     />
     <link rel="stylesheet" href="css/style.css" />
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <title>Mathsli</title>
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <header>
+     <h1>Mathsli</h1> 
+     <h3>Do you love Maths? </h3>
+     <h5>Or Maybe...?</h5>
+     <p>Here is the right place for you to be anyway.</p>
+     <nav>
+      <ul class="list">
+          <li class="items"><a class="links" href="#"> About Mathli</a></li>
+          <li class="items"><a class="links" href="#"> Testimonials</a></li>
+          <li class="items"><a class="links" href="#"> Contact</a></li>
+          <li class="items"><a class="links" href="#"> Login</a></li>  
+      </ul>     
+  </nav>
+    </header>
+    <main>
+      <section>
+      <article class="article1">
+        <h1>About Mathsli</h1>
+        <h4>Mathsli helps you to learn and love Maths.</h4>
+    <img src="https://clipartion.com/wp-content/uploads/2015/11/what-are-the-top-biggest-brain-damaging-habits-magical-maths.jpg" 
+     alt="">
+    <h2>
+      We are here to help.
+    </h2>
+    <button><a href="#">More About US</a></button>
+    <h3>
+      All our teachers are professionals with strong academic backgrounds who 
+      enjoy helping students understand concepts deeply, solving challenging questions and also familiar them with 
+      the beauty of mathematics. 
+    </h3> 
+    <h4> Thank you for visiting our website. 
+      Please write down any comment you have in the box bellow.
+    </h4> 
+      </article>
+      <section>
+    </main>
+    <textarea name="comment"
```

Comment:
Every form element should have a `label` associated with it, to help screen reader users. A placeholder isn't a good substitute as it disappears when the user clicks on the input box.